specification/webpubsub/resource-manager/Microsoft.SignalRService/stable/2021-10-01/webpubsub.json

```diff
@@ -0,0 +1,2914 @@
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "202": {
+            "description": "Accepted. The response indicates the delete operation is performed in the background."
+          },
+          "204": {
+            "description": "Success. The response indicates the event handler is already deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "WebPubSubEventHandlers_Delete": {
+            "$ref": "./examples/WebPubSubEventHandlers_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/webPubSub/{resourceName}/hubs/{hubName}/eventHandlers/eventHandlerName": {
+      "get": {
+        "tags": [
+          "WebPubSub"
+        ],
+        "description": "Get an event handler.",
+        "operationId": "WebPubSubEventHandlers_Get",
+        "parameters": [
+          {
+            "name": "hubName",
+            "in": "path",
+            "description": "The hub name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "eventHandlerName",
+            "in": "query",
+            "description": "The event handler name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response describe the corresponding event handler.",
+            "schema": {
+              "$ref": "#/definitions/EventHandler"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WebPubSubEventHandlers_Get": {
+            "$ref": "./examples/WebPubSubEventHandlers_Get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/webPubSub/{resourceName}/hubs/hubName": {
+      "get": {
```

Comment:
>get":  [](http://example.com/codeflow?start=7&length=6)

This should move to '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/webPubSub/{resourceName}/hubs/{hubName}' above.